src/showlib/processing/l0_to_l1a.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+from skretrieval.util import configure_log
+
+from showlib.flights.er2_2023.Platform.ER2Platform import ER2Platform as er2
+from showlib.flights.er2_2023.Specifications.shower2_specs import SHOW_specs as specs
+from showlib.l1a.data import L1AFileWriter, L1AImage
+from showlib.l1a.l1a_processing import level1A_processing as l1a
+
+
+def er2_attitude(show_utc, filename):
+    er2data = xr.load_dataset(filename)
+    slice = er2data.sel(time=show_utc, method="nearest")
+
+    er2altitude = slice.altitude.data
+    pitch = slice.pitch.data
+    roll = slice.variables["roll"].data
+    heading = slice.heading.data
+    latitude = slice.latitude.data
+    longitude = slice.longitude.data
+
+    observer = [latitude, longitude, er2altitude]
+    aircraft_orientation = [heading, pitch, roll]
+
+    return observer, aircraft_orientation
+
+
+def geometry_setup(
+    shs_config, show_utc_time, observer=None, aircraft_orientation=None
+):
+    #######################################################Get the sampling information###########################################
+    if aircraft_orientation is None:
+        aircraft_orientation = [45, 0, 0]
+    if observer is None:
+        observer = [0, 135, 21000]
+    ER2geometry = er2(shs_config)
+    ER2geometry.make_ER2_geometry(
+        utc=show_utc_time, observer=observer, aircraft_orientation=aircraft_orientation
+    )
+    return ER2geometry
+
+
+def set_measurement_config(
+    utc=None,
+    iWGnc_filename=r"\\datastore\valhalla\root\arg_measurements\SHOW\2023_HAWC-ER2_Palmdale\attitude\iwg1-50hz-27Nov2023-2214.nc",
+):
+    # SHS configuration
+    shs_config = specs(littrow_nm=1e7 / (7334.808944166525), M=0.2196762889195097)
```

Comment:
can this be put somewhere else? The calibration DB?

Reply:
added the littrow and M to the spec.yaml